app/models/product.rb

```diff
@@ -10,10 +10,11 @@ class Product < ApplicationRecord
   validates :gender,      presence: true
   validates :quantity,    presence: true
 
-
   enum gender: {
     "Female": 0,
     "Male": 1
   }
 
+  scope :premium, -> { where("price > ?", 3000) }
```

Comment:
Hey Sarine! Sorry just to check. Would this work as where(price > 3000)? Does the ? have to be included in the query?